components/collector/src/source_collectors/jenkins/job_runs_within_time_period.py

```diff
@@ -36,4 +36,4 @@ def _include_build(self, build: Build) -> bool:
         result_types = [result_type.lower() for result_type in self._parameter("result_type")]
         lookback_days = int(cast(str, self._parameter("lookback_days")))
         build_datetime = datetime_from_timestamp(int(build["timestamp"]))
-        return days_ago(build_datetime) <= lookback_days and build["result"].lower() in result_types
+        return days_ago(build_datetime) <= lookback_days and build.get("result", "").lower() in result_types
```

Comment:
Is this the only place where we get `build["result"]`? I'd expect it to be in multiple jenkins collectors

Reply:
We already did `build.get("result")` in other locations. This was the only place where it was `build["result"]`.